src/components/ProgramDrawer.tsx

```diff
@@ -154,22 +154,203 @@ function EQIPCheckboxList({ setEQIPChecked, setShowPopUp }) {
     );
 }
 
-export default function ProgramDrawer({ setEQIPChecked }): JSX.Element {
+function CSPCheckboxList({ setCSPChecked, setShowPopUp }) {
+    const [checked, setChecked] = React.useState(currentChecked);
+
+    const handleToggle = (value: number) => () => {
+        setChecked(value);
+        setCSPChecked(value);
+        currentChecked = value;
+        setShowPopUp(false);
+    };
+
+    const CSPList = [
+        "Total CSP Benefits",
+        "Land management",
+        "Other improvement",
+        "Existing activity payments",
+        "Vegetative",
+        "Forest management",
+        "Soil remediation",
+        "Structural",
+        "Bundles",
+        "Cropland",
+        "Rangeland",
+        "Pastureland",
+        "Other: supplemental, adjustment & other",
+        "NIPF",
```

Comment:
```suggestion
        "Non-industrial private forestland",
```
A couple of weeks ago, Jonathan had suggested expanding the abbreviations in a direct Slack conversation related to CSP data.

Reply:
Ah I don't know this, I will change

Reply:
Thanks.